src/docs/product/releases/health.mdx

```diff
@@ -48,15 +48,15 @@ Hover over specific dates and times in the overview graph for a quick perspectiv
 
 Crashed
 
-: The app had an explicit unhandled error or hard crash. You'll typically be able to see the corresponding issue in Sentry that captures this event.
+: The app had an explicit unhandled error or hard crash. You'll typically be able to see the corresponding issue in Sentry that captures this event. To search for them in _Discover_ or _Issue Stream_, filter by `handled:no`.
```

Comment:
```suggestion
: The app had an explicit unhandled error or hard crash. You'll typically be able to view the corresponding issue in Sentry that captures this event. To search for them in **Discover** or on the **Issues** page, filter by `handled:no`.
```

Reply:
Can we specify in some style guide which style we want to use? I'm still following the conventions put forward in https://github.com/getsentry/sentry-docs/issues/441, which are applied throughout most of docs.